atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -251,8 +252,12 @@ def set_group_manifest_info(self, extra, worker_metadatas):
                 version_release = image.tag
                 break
 
-        assert version_release is not None, 'Unable to find version-release image'
-        tags = [image.tag for image in primary_images]
+        if is_scratch_build():
+            tags = [image.tag for image in self.workflow.tag_conf.images]
+            version_release = tags[0]
+        else:
+            assert version_release is not None, 'Unable to find version-release image'
```

Comment:
Why this assert? (Should it be really assert)

Reply:
that was there already before, well it shouldn't ever happen

Reply:
Ah sorry missed that